src/guides/wiki/oba-2x-config-deploy-guide.md

```diff
@@ -5,37 +5,39 @@ layout: page
+
+Note that unless there's a good reason to configure and deploy manually, the official Docker image should be used.
 
 ## Note on Software Versions
 
 When using this guide, be certain to use the exact same versions of all software mentioned here. Any discrepancies between this guide and your actual installation will more than likely result in complicated errors and, ultimately failure. Follow the instructions exactly and there should be no issues.
 
 ## Server Minimum Requirements
 
-The minimum system requirements for your server depend on the size of the transit agency that the deployment is for. Failure to select a server with sufficient processing power and, more importantly, memory, will result in the Tomcat service, or the server itself needing a reboot several times a day. This guide is written for a small to medium sized municipal transit agency, with about 64 routes, 317 vehicles and 2414 stops. For this deployment I am using a virtual server with 2.0 GHz of processor power and 2048 MB of memory.
+The minimum system requirements for your server depend on the size of the transit agency that the deployment is for. Failure to select a server with sufficient processing power and, more importantly, memory, will result in the Tomcat service or the server itself needing a reboot several times a day. This guide is written for a small to medium sized municipal transit agency, with about 64 routes, 317 vehicles and 2414 stops. In this guide's deployment, a virtual server with 2.0 GHz of processor power and 2048 MB of memory is being used.
 
 ## Server Operating System
-This guide assumes the end user already knows how to set up a server (virtual or dedicated) using the Debian 9 operating system. We need to start with a clean installation of Debian 9 with no extra anything, not even the Debian desktop environment.
+This guide assumes the end user already knows how to set up a server (virtual or dedicated) using the Debian 9 operating system. You need to start with a clean installation of Debian 9 with nothing extra, not even the Debian desktop environment.
 
-When setting up Debian 9 using the installer, be certain not to install any additional components except for the SSH server and standard system utilities. You can probably get away with including more on the server but this guide is written with a clean, lightweight server environment in mind and making your server the same will help insure success. Also, the less that is running on the server, the more resources available for OneBusAway.
+When setting up Debian 9 using the installer, be certain to not install any additional components except for the SSH server and standard system utilities. You can likely include more on the server but this guide is written with a clean, lightweight server environment in mind and making your server the same will help ensure success. Also, the less that is running on the server, the more resources available for OneBusAway.
```

Comment:
Debian 9 hasn't received any security updates for almost two years. This guide should be updated to use either:

* Debian 12, which will receive security updates until 2028. https://wiki.debian.org/LTS
* Ubuntu 22.04, which is what the docker image is using, and is also an LTS release

Reply:
Gotcha, thanks. I'll make those changes and then submit a new PR with all the commits squashed properly.

Reply:
You can keep the same PR. All of the commits can be squashed in here before merge